e2e-tests/micronaut-rest-api/src/test/java/com/foo/micronaut/rest/MicronautTestController.java

```diff
@@ -0,0 +1,83 @@
+package com.foo.micronaut.rest;
+
+import org.evomaster.client.java.controller.EmbeddedSutController;
+import org.evomaster.client.java.controller.InstrumentedSutStarter;
+import org.evomaster.client.java.controller.api.dto.AuthenticationDto;
+import org.evomaster.client.java.controller.api.dto.SutInfoDto;
+import org.evomaster.client.java.controller.internal.SutController;
+import org.evomaster.client.java.controller.problem.ProblemInfo;
+import org.evomaster.client.java.controller.problem.RestProblem;
+
+import java.sql.Connection;
+import java.util.List;
+
+public class MicronautTestController extends EmbeddedSutController {
+
+    private MicronautApplication application;
+
+    public MicronautTestController() {}
+
+    @Override
+    public String startSut() {
+        application = new MicronautApplication(9000);
```

Comment:
better to use a random port instead of fixed one.
In addition, add a method `getSutPort` to get the specific port once the application is started (could check this https://github.com/EMResearch/EvoMaster/blob/master/e2e-tests/spring-rest-openapi-v2/src/test/java/com/foo/rest/examples/spring/SpringController.java)